app/assets/stylesheets/apply.scss.erb

```diff
@@ -56,34 +97,114 @@
     font-weight: 400;
 }
 
-.decision-criteria {
-  padding-left:50px;
-  p, h4 {
-    text-align:left;
+#hook {
+  text-align: left;
+}
+
+#apply-button {
+  text-align: center;
+}
+
+.submenu {
+  list-style:none;
+  font-size: 0.9em;
+  font-weight: bold;
+  text-align: center;
+
+  li {
+    display: inline-block;
+    padding: 5px 10px;
+    margin: 0;
+    border-left: 1px solid #AAA;
+    border-bottom: 5px solid rgba($blue, 0);
+    transition-duration: 0.3s;
+    cursor: pointer;
+
+    &:first-child {
+      border: none;
+    }
+
+    &:hover {
+      background-color: #DDDDDD;
+      border-bottom: 5px solid rgba($blue, 1);
+    }
   }
 }
 
-.blue-block {
-  padding: 25px 10px;
-  background-color: $blue;
-  color:#fff;
-  &.text-center {
-    text-align:center;
+.title-column {
+  @extend .large-3;
+  @extend .medium-4;
+  @extend .small-12;
+  @extend .columns;
+  text-align: right;
+
+  h1 {
+    text-align: right;
+  }
+
+  @media only screen and (max-width: em(640)) {
```

Comment:
If you uncomment the `@import "grid-settings";` line in `stylesheets/base/_base.scss` then you can just use

    @include media($medium-screen) {
        text-align ...
    }